Dbal/AclProvider.php

```diff
@@ -225,13 +227,12 @@ public function findAcls(array $oids, array $sids = array())
      * ACEs, and security identities.
      *
      * @param array $ancestorIds
+     * @param array $identityIds
      *
      * @return string
      */
-    protected function getLookupSql(array $ancestorIds)
+    protected function getLookupSql(array $ancestorIds, array $identityIds)
```

Comment:
This is a BC break, you should use a trick like this: https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Serializer/Serializer.php#L201-L214